easybuild/easyconfigs/b/BEDTools/BEDTools-2.31.0-GCC-12.3.0.eb

```
@@ -0,0 +1,53 @@
+# Author: Maxime Schmitt, University of Luxembourg
+# Author: Adam Huffman, The Francis Crick Institute
+#
+# Based on the work of: Pablo Escobar Lopez
+# Swiss Institute of Bioinformatics (SIB)
+# Biozentrum - University of Basel
+
+easyblock = 'MakeCp'
+
+name = 'BEDTools'
+version = '2.31.0'
+
+homepage = 'https://bedtools.readthedocs.io/'
+description = """BEDTools: a powerful toolset for genome arithmetic.
+The BEDTools utilities allow one to address common genomics tasks such as finding feature overlaps and
+computing coverage.
+The utilities are largely based on four widely-used file formats: BED, GFF/GTF, VCF, and SAM/BAM."""
+
+toolchain = {'name': 'GCC', 'version': '12.3.0'}
+
+source_urls = ['https://github.com/arq5x/bedtools2/archive/refs/tags/']
+sources = ['v%(version)s.tar.gz']
+checksums = ['183cf9a96aabc50ef4bd557a53fd01557a123c05a0dc87651371878f357439ec']
+
+builddependencies = [
+    ('Python', '3.11.3'),
+]
+dependencies = [
+    ('XZ', '5.4.2'),
+    ('zlib', '1.2.13'),
+    ('bzip2', '1.0.8'),
+    ('BamTools', '2.5.2'),
+]
+
+buildopts = 'CXX="$CXX"'
+
+files_to_copy = [
+    'bin',
+    'docs',
+    'data',
+    'genomes',
+    'scripts',
+    'test',
+]
+
+sanity_check_paths = {
+    'files': ['bin/%s' % x for x in ['bedtools', 'pairToBed', 'mergeBed', 'bedToBam', 'fastaFromBed']],
+    'dirs': ['bin', 'docs', 'data', 'genomes', 'scripts', 'test'],
```

Could we here re-use the array defined above?
```suggestion
    'dirs': files_to_copy,
```